hera_cal/datacontainer.py

```diff
@@ -84,12 +92,80 @@ def __delitem__(self, key):
         else:
             raise ValueError('only supports setting (ant1, ant2, pol) keys')
 
+    def concat(self, D, axis=0):
```

Comment:
To match numpy, I'd call this `concatenate`

Reply:
sure